src/index.js

```diff
@@ -0,0 +1 @@
+export default () => 'Welcome to perf-goggles';
```

Comment:
Interesting. Is this added by default by the Neutrino wizard?

Reply:
Yes! Would you recommend removing this file?